doc/cephfs/administration.rst

```diff
@@ -92,6 +92,14 @@ The CephX IDs authorized to the old file system name need to be reauthorized
 to the new name. Any on-going operations of the clients using these IDs may be
 disrupted. Mirroring is expected to be disabled on the file system.
 
+::
+
+    fs swap <fs1-name> <fs2-name> [--swap-fscids] [--yes-i-really-mean-it]
+
+Swaps names of two Ceph file sytems and updates the application tags on all
+pools of both FSs accordingly. Passing ``swap-fscids`` swaps FSCIDs as well.
+Mirroring should be disabled on both the FSs. CephX IDs needs to reauthorized
+accordingly. On-going operations using these IDs may be disrupted.
```

Comment:
Also why

Reply:
Why for? "CephX IDs needs to reauthroized"?

Reply:
Yes. If the `fs rename` docs explain, refer to that. Otherwise both should reference some explanation.

Reply:
ping

Reply:
For something like this should we also give a warning or hint after the `fs swap` succeeding ?